tests/libvmi/cloudinit.go

```diff
@@ -86,16 +86,27 @@ func addDiskVolumeWithCloudInitConfigDrive(vmi *v1.VirtualMachineInstance, diskN
 	addDisk(vmi, newDisk(diskName, bus))
 	v := newVolume(diskName)
 	v.VolumeSource = v1.VolumeSource{CloudInitConfigDrive: &v1.CloudInitConfigDriveSource{}}
-	addVolume(vmi, v)
+	cloudInitVolumeExists(vmi, v)
```

Comment:
I don't understand what this function does.
The returned value is not used and the volume is not added.
Maybe something is missing here? :)

Reply:
Please have another look :slightly_smiling_face: